### PR TITLE
zh-cn: init translation of cycletracker manifest

### PR DESCRIPTION
**Pull request by hanyujie2002**
### Description
Same as title

### Motivation
ditto

### Additional details
None

### Related issues and pull requests

Related to #15548

### CODE REVIEW

**Comment by yin1999** [comment]
One suggestion, could you setup node runtime locally and install pre-commit hook for this repo (run `yarn install` so the pre-commit hook would be installed)?

**Comment by hanyujie2002** [comment]
> One suggestion, could you setup node runtime locally and install pre-commit hook for this repo (run `yarn install` so the pre-commit hook would be installed)?

I am sorry, It told me to install visual studio but I don't want to install it😢

image

**Comment by yin1999** [comment]
> > One suggestion, could you setup node runtime locally and install pre-commit hook for this repo (run `yarn install` so the pre-commit hook would be installed)?
> 
> I am sorry, It told me to install visual studio but I don't want to install it😢
> 
> image
> Thank you for your suggestion and I will consider it sincerely 😁

You could try to install [VS build tools](https://visualstudio.microsoft.com/zh-hans/downloads/#build-tools-for-visual-studio-2022)

**Comment by hanyujie2002** [comment]
> > > One suggestion, could you setup node runtime locally and install pre-commit hook for this repo (run `yarn install` so the pre-commit hook would be installed)?
> > 
> > 
> > I am sorry, It told me to install visual studio but I don't want to install it😢
> > image
> > Thank you for your suggestion and I will consider it sincerely 😁
> 
> You could try to install [VS build tools](https://visualstudio.microsoft.com/zh-hans/downloads/#build-tools-for-visual-studio-2022)

Thank you, I will have a try

File: files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/manifest_file/index.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
+---
+title: CycleTracker：清单和图标
```
Comment (yin1999) [inline files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/manifest_file/index.md:2]:
```suggestion
title: 经期跟踪器：清单和图标
```

File: files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/manifest_file/index.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+      "sizes": "512x512"
+    }
+  ],
+  "display": "standalone"
+}
+```
+
+在保存清单文件并从我们的 HTML 文件中链接到它之前，我们可以开发一个仍然简洁但具有更多信息的 JSON 对象，以定义 PWA 的身份、展示和图标。是的，上面的内容会起作用，但让我们讨论一下这个示例中的成员以及其他一些成员，这些成员使清单文件能够更好地定义我们的经期跟踪器 PWA 的外观。
+
+## 应用身份
```
Comment (yin1999) [inline files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/manifest_file/index.md:32]:
```suggestion
## 应用标识
```

**Review by yin1999 — APPROVED** [review]
Thanks @hanyujie2002 👍. I've pushed a commit to align the translation of title

**Pull request merged**